File: letcode-26.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums:
            return 0
        s = 0
        for e in range(1, len(nums)):
            if nums[s] != nums[e] and e - s > 1:
                s += 1
                nums[s] = nums[e]
        return s + 1


def removeDuplicates(nums: List[int]) -> int:
    if not nums:
        return 0
    s = 0
    for e in range(1, len(nums)):
        if nums[s] != nums[e]:
            if e - s > 1:
                nums[s + 1] = nums[e]
            s += 1
    return s + 1


print(removeDuplicates([1, 2]))

File: letcode-26.go
package main

func removeDuplicates(nums []int) int {
	f:=0
	for i:=1;i<len(nums);i++{
		if nums[f]!=nums[i] {
			if i-f>1{
				nums[f+1]=nums[i]
			}
			f++
		}

	}
	return f+1
}
